popup/popup.js

```diff
@@ -1,7 +1,7 @@
 
 window.onload = function() {
     document.getElementById('closeBtn').onclick = closeExtension;
-    document.getElementById('searchField').addEventListener('keyup', updateLocalStorage);
+    document.getElementById('searchField').addEventListener('input', updateLocalStorage);
```

Comment:
I believe we should change `input` to `change`. This stops it from updating the local storage with every added character. It also has all the correct behaviours, like when closing the extension everything saves correctly.